pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mkdocs-drawio-file"
-version = "1.3.0"
+version = "1.5.0"
```

Comment:
@onixpro this time I increased the version in project.toml as well. This should fix the build and deployment of the plugin on pypi :)